frontend/src/component/project/Project/ProjectHealth/ReportTable/ReportTable.tsx

```diff
@@ -145,7 +144,7 @@ export const ReportTable = ({ projectId, features }: IReportTableProps) => {
                 maxWidth: 120,
             },
         ],
-        [showEnvironmentLastSeen]
+        [showEnvironmentLastSeen, JSON.stringify(data)]
```

Comment:
Is JSON.stringify(data) really needed? I guess this is to make it change if anything on the response changes, but it feels excessive

Reply:
removed